src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1535,23 +1540,23 @@ class Simplex_tree {
    * and computes the extended persistence diagram induced by the lower-star filtration 
    * computed with these values. 
    * \post Note that after calling this function, the filtration 
-   * values are actually modified. The function `compute_extended_persistence_subdiagrams()` 
+   * values are actually modified. The function `extended_persistence_subdiagrams()` 
    * retrieves the original values and separates the extended persistence diagram points 
    * w.r.t. their types (Ord, Rel, Ext+, Ext-) and should always be called after 
    * computing the persistent homology of the extended simplicial complex.
    * \pre Note that this code creates an extra vertex internally, so you should make sure that
    * the Simplex tree does not contain a vertex with the largest Vertex_handle. 
    */
-  void extend_filtration() {
+  Extended_filtration_data extend_filtration() {
```

Comment:
In the doc, mention that it returns information needed by the other function.

Reply:
Done.